skorch/callbacks/lr_scheduler.py

```diff
@@ -142,6 +137,32 @@ def on_train_begin(self, net, **kwargs):
             net, self.policy_, **self.kwargs
         )
 
+    def _step(self, net, lr_scheduler, score=None):
+        """Helper method to step the lr scheduler.
+
+        This takes care of two things:
+
+        1. If the lr scheduler is ReduceLROnPlateau, we need to pass the score.
+        2. If the net is uses AccelerateMixin, stepping has to be skipped in
+           certain conditions.
+
+        For more info on the latter, see:
+        https://huggingface.co/docs/accelerate/quicktour#mixed-precision-training
+
+        """
+        accelerator_maybe = getattr(net, 'accelerator', None)
```

Comment:
Yes, I wanted (and succeeded so far) to avoid it, but here, I think this might be the best solution. 

Alternatively, we could have a method on the net that returns `False` by default and proxies `accelerator_maybe.optimizer_step_was_skipped` on `AccelerateMixin`, do you think that's better?

Reply:
> do you think that's better?

I do not think it's better. I'm okay with keeping the current version.